.env.example

```diff
@@ -114,3 +114,5 @@ GREENHOUSE_USER_ID=123456789
 GREENHOUSE_API_TOKEN=xxx
 
 BOKUN_ACCESS_TOKEN=xxx
+
+IP2LOCATION_API_KEY=xxx
```

Comment:
nitpick:
- please, keep the file sorted by alphabet
- keep newline at the end of file (we should really add .editorconfig)

Reply:
>please, keep the file sorted by alphabet

Would preferably open a separate PR for that, since the file is very un-alphabetical even aside from what I just added